lib/Doctrine/ORM/Cache/CacheConfiguration.php

```diff
@@ -1,5 +1,8 @@
 <?php
 
+declare(strict_types = 1);
```

Comment:
Just a note on style: the `declare(strict_types=1)` (no spaces) should be after the licence